ProjectTemplates/AspNetCoreMicrosoftIdentityPlatformTemplates.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageType>Template</PackageType>
+    <PackageVersion>0.1.0</PackageVersion>
+    <PackageId>Microsoft.IdentityPlatform.Templates</PackageId>
+    <Title>ASP.NET Core Web app and Web API templates with Microsoft identity platform</Title>
+    <Authors>Microsoft</Authors>
+    <Description>
+      Templates to use when creating a web app that signs in users or a protected Web API
```

Comment:
nit: one template or two templates? sincerely asking.

Reply:
the web app template will come soon :)
you are right for the moment, there is only one (there was 2 yesterday, but I remove an obsolete one)